src/compositor.rs

```diff
@@ -1122,20 +1158,34 @@ impl IOCompositor {
     fn remove_webview(
         &mut self,
         top_level_browsing_context_id: TopLevelBrowsingContextId,
-        window: &mut Window,
+        windows: &mut HashMap<WindowId, Window>,
     ) {
-        debug!("{}: Removing", top_level_browsing_context_id);
-        let Some(webview) = window.remove_webview(top_level_browsing_context_id, self) else {
-            warn!("{top_level_browsing_context_id}: Removing unknown webview");
-            return;
-        };
+        debug!(
+            "Verso Compositor is removing webview {}",
+            top_level_browsing_context_id
+        );
+        let mut window_id = None;
+        for window in windows.values_mut() {
+            let (webview, close_window) =
+                window.remove_webview(top_level_browsing_context_id, self);
+            if let Some(webview) = webview {
+                self.set_painting_order(window);
```

Comment:
`set_painting_order(window);` might not run if the window is not the current window. Is this expected?

Reply:
yes, this is intended. If this is not the current window, we don't need to update the painting order.